config/application.rb

```diff
@@ -21,15 +24,24 @@ module LinkCheckerApi
   self.hosts_with_basic_authorization = {}
 
   class Application < Rails::Application
-    # Settings in config/environments/* take precedence over those specified here.
-    # Application configuration should go into files in config/initializers
-    # -- all .rb files in that directory are automatically loaded.
-    config.load_defaults = 6.0
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 5.0
```

Comment:
Weirdly `config.load_defaults` was set to 5.0 when running app:update after updating the gemfile to 6.1 🤷‍♀️ 

I manually changed it to 6.0 and ran the tests, but they continued to fail

Reply:
I've come across this before too 🤔 Not sure why!